src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx

```diff
@@ -53,6 +54,55 @@ export const CreateQuizSecondCard: React.FC = observer(() => {
     setAnswerList(newList);
   };
 
+  const handleDropdownChange = (value: any) => {
+    // When changing dropdown value, check for what action to take
+    formik.setFieldValue("questionType", value.value);
+    if (formik.values?.questionType !== QUESTION_TYPES.SUBJECTIVE.value) {
+      // Reset the correctAnswers when changing question type
+      setAnswerList((currentState) => {
+        let arrayCopy = [...currentState];
+        return (arrayCopy = arrayCopy.map((answer, index) => {
```

Comment:
There is no need to assign the result of map to `arrayCopy` as it is not used anymore later on. In this case, directly returning the result of map is enough.